File: Week1/Pangrams/src/main/java/MyCode.java
package main.java;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

public class MyCode {
    List<Character> alphabet = Arrays.asList(
            'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
            'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'
    );
    public static String pangrams(String s) {
        List<Character> letters = new ArrayList<Character>();
        for (char letter : s.toLowerCase().toCharArray()){
            if(letter != ' '){
                letters.add(letter);
            }
        }
        if( (letters.stream().collect(Collectors.toSet()).size()) == 26){
            return "PANGRAM";
        } else {
            return "NOT PANGRAM";
        }
    }
}

File: Week1/Pangrams/src/main/kotlin/MyCode.kt
package main.kotlin

fun pangrams(s: String):String{
    val string = s.replace("\\s".toRegex(), "").toLowerCase().toSet()

    return if( string.size == 26) "pangram" else "not pangram"
}
fun main(args: Array<String>) {
    val s = readLine()!!

    val result = pangrams(s)

    println(result)
}